src/vector/index.html

```diff
@@ -28,7 +28,7 @@
         style-src 'self' 'unsafe-inline' <%= csp_extra_source %>;
         script-src 'self' 'wasm-unsafe-eval' https://www.recaptcha.net/recaptcha/ https://www.gstatic.com/recaptcha/ <%= csp_extra_source %>;
         img-src * blob: data:;
-        connect-src *;
+        connect-src * blob: data:;
```

Comment:
This likely has security implications - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/connect-src - it might create an XSS vulnerability around anchors pointing at executable blobs

Reply:
But this is required to retrieve the data from script... No chance to do this? 

And, I can't understand your XSS vulnerability and I can't find anything about this in the link provided :( My poor knowledge about web 🤣